CHANGELOG.md

```diff
@@ -50,6 +50,9 @@ CLI command and its behaviour. There are no guarantees of stability for the
 ### Changed
 
 - Alpine Docker image now uses 3.18 as base. (#846)
+- No longer create and publish `-extra` Docker images. (#849)
```

Comment:
Could this even be breaking?

Reply:
And shall we also mention that the `dev` images will not be supported any more?

Reply:
I'm inclined to not treat it as breaking. Nothing breaks; the images will keep working. It's just that no more -extra images will be published.

Added a line for `dev`. Removing `dev` _might_ be a breaking change, but I'm not sure it would matter much whether this is communicated during a minor version or a major version. But honestly, people should expect `dev` to break _anyway_.

Reply:
OK. Perhaps we should remember ourselves to delete some of these tags altogether so that users know that something broke. Keeping outdated, unversioned tags (like `latest-extra`) around seems like a bad practice.

Reply:
Good call, created #850. Thanks a heap for your assistance @mxmehl !